sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -1064,6 +1064,24 @@ void CAFMaker::produce(art::Event& evt) noexcept {
     }
   }
 
+  // Get all of the OpFlashes
+  std::vector<caf::SROpFlash> srflashes;
+
+  for (unsigned i_tag = 0; i_tag < pandora_tag_suffixes.size(); i_tag++) {
+    const std::string &pandora_tag_suffix = pandora_tag_suffixes[i_tag];
+    art::Handle<std::vector<recob::OpFlash>> flashes_handle;
+    GetByLabelStrict(evt, fParams.OpFlashLabel() + pandora_tag_suffix, flashes_handle);
+    // fill into event
+    if (flashes_handle.isValid()) {
+      const std::vector<recob::OpFlash> &opflashes = *flashes_handle;
+      int cryostat = ( pandora_tag_suffix.find("W") != std::string::npos );
```

Comment:
Gross, but presumably a pre-existing pattern. Can you at least put a `? 1 : 0;` on the end?

Reply:
Yeah, I agree this is gross. Happy to add that if it makes it look prettier, but if there's a more clever way of picking out the cryostat I am all ears.